server/src/main/java/org/apache/iotdb/db/metadata/idtable/entry/DeviceEntry.java

```diff
@@ -32,9 +32,18 @@
 
   boolean isAligned;
 
+  // for manages flush time
+  Map<Long, Long> lastTimeMap;
+
+  Map<Long, Long> flushTimeMap;
```

Comment:
Why not put both two maps in IDTableFlushTimeManager, like what LastFlushTimeManager does? I think it's IFlushTimeManager's responsibility to maintain both maps.

Reply:
There is another class implements ILastFlushTimeManager called LastFlushTimeMananger which uses map to maintain flush time of each device. However, if we enable id table, id table has a map which key is device ID and value is device entry. So there is no need to hold another map keyed by device. We can manage our flush time in that device entry. So I let DeviceEntry hold these 2 flush time maps. To sum up, this implementation can save memory by using id table's map rather than an redundant device map.